src/app/clusters/commissioner-control-server/commissioner-control-server.cpp

```diff
@@ -216,6 +216,14 @@ bool emberAfCommissionerControlClusterCommissionNodeCallback(
 
     auto sourceNodeId = GetNodeId(commandObj);
 
+    // Constraint on responseTimeoutSeconds is [0; 120] seconds
+    if ((commandData.responseTimeoutSeconds < 30) || (commandData.responseTimeoutSeconds > 120))
```

Comment:
Either comment is wrong or `responseTimeoutSeconds < 30` condition.

Reply:
Oops ... comment is wrong. Will fix it.